install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -54,7 +54,11 @@ data:
         restartPolicy: Always
       containers:
       - name: istio-proxy
-        image: {{ .Values.global.hub }}/{{ .Values.global.proxy.image }}:{{ .Values.global.tag }}
+        image: {{ "{{ if (isset .ObjectMeta.Annotations \"sidecar.istio.io/proxyImageOverride\") -}}" }}
```

Comment:
Should we use a different deliminator for runtime template, e.g. `[[` and `]]` instead of `{{` and `}}`. We already do this in mesh.go to make it clearer which template "level" we're operating at.

Reply:
We can only change the inner template of Helm template. So we also need to swap the use of delimiter in mesh.go to use [[ for inner templates while {{ for outer ones, right?

Reply:
Yup, exactly. See https://golang.org/pkg/text/template/#Template.Delims).